### DISCUSSION TITLE
Unable to claim Github Student pack azure subscription

### DISCUSSION THREAD

**Discussion by ChethanHR100** [community/community#103110] [GitHub Education]
### Select Topic Area

Question

### Body

I verified my student status and got Github Student Dev pack, which also contains an Azure student subscription. But am unable to access the Azure student subscription.
![Screenshot (219)](https://github.com/community/community/assets/155728900/c164d61a-edef-4e5b-bd4c-65f40669a48c)

**Comment by gil-son** [comment]
You must to contact [Github Support](https://support.github.com/)

They will check it for you

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @ChethanHR100! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
